Insurance_company/insurance_app/models.py

```diff
@@ -46,6 +55,28 @@ def __str__(self):
         return f"Nazwa polisy: {self.policy_name}"
 
 
+class HouseInsurance(models.Model):
+
+    HOUSE_TYPES = (
+        (1, "Dom"),
+        (2, "Szeregowiec"),
+        (3, "Mieszkanie")
+    )
+
+    # predefined fields
+    policy_id = models.CharField(generate_id(), primary_key=True)
+    policy_name = models.CharField(max_length=100)
+    policy_description = models.TextField(max_length=2000)
+    valid_to = models.DateField(auto_now=True)
+    price = models.DecimalField(max_digits=8, decimal_places=2, default=0.00)
```

Comment:
czy to nie jest bardziej value, niż price? Mała rzecz, ale chyba chodzi o wartość domu, a nie jego cenę

Reply:
@adam-harmasz tutaj miałem na myśli pole na cenę polisy, która będzie obliczać się po wypełnieniu innych danych przez użytkownika. Natomiast osobno stworzę pole z wartością nieruchomości